CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing
+
```

Comment:
can we update the intro section to be similar to these

https://github.com/coinbase/onchainkit/blob/main/CONTRIBUTING.md
https://github.com/base/web/blob/master/CONTRIBUTING.md